lens_v2/package.json

```diff
@@ -2,12 +2,13 @@
   "name": "lens-subgraph",
   "license": "MIT",
   "scripts": {
-    "codegen": "graph codegen",
-    "build": "graph build",
+    "codegen": "npx @airstack/subgraph-generator social --version v2.0.0 --slug lens_v2_polygon --name lens && graph codegen",
+    "build": "npm run codegen & graph build",
+    "graph:build": "graph build", 
     "test": "graph test"
```

Comment:
deployment has steps 
```       
npm run codegen
npm run build
npm run deploy
```
in [push-subgraph.yml](https://github.com/Airstack-xyz/Subgraphs/blob/main/.github/workflows/push-subgraph.yml)

we should modify either this or deployment file

Reply:
Done